src/less/button/base/button.less

```diff
@@ -1,6 +1,8 @@
 @import "../../less/variables-common.less";
 @import "../../less/mixins-common.less";
 
+/* stylelint-disable  no-descending-specificity */
```

Comment:
I assume we're just punting specificity issues related to these files until we can rework things?

Reply:
Yep, I did mention that in the description. The more complex ones we need to come back to separately.

Reply:
👍

Reply:
My next PR will go through the simple stuff from G through Z (a bit like this PR). Then I will enable the stylelint config rule. Then we will start picking off the complex ones.